Restyling "How to run access-om2" page + Markdown restyling

atteggiani opened the pull request:
The recently updated page "How to run ACCESS-OM2" had a few styling and formatting issues, that were making it not consistent with other pages in the website.

**MODIFICATIONS SPECIFIC TO THE PAGE**
- [x] Reformat "References" using HTML to have the same styling as other pages in the website;
- [x] A few rewording/additions/deletions (The structure of the page was left untouched, with "how to run" section coming before "how to modify")

**MODIFICATIONS NOT SPECIFIC TO THE PAGE**
- [x] Modified styling guidelines in the README
- [x] Changed CSS so that admonitions (`!!! ...` and `??? ...` in markdown) have the desired style.
From now on the admonitions can be used as described in the [admonition documentation](https://squidfunk.github.io/mkdocs-material/reference/admonitions/), with a few differences in the rendered style.
Therefore, markdown syntax is encouraged for readability. 
More information about styling guidelines in the README.
- [x] Other minor modifications 

I have converted everything to Markdown.
As you can see it's all Markdown, with the exception of virtual terminal windows and references (in the near future we will style references properly also in Markdown).

Feel free to comment on any wording/addition/deletion, or any other part you think needs changing.

**Note**:
You will see a lot of files changed listed. Most of them are inside the `fontello` folder, for icons. 
The main file changed is the `run-access-om.md`. 

Cheers

aidanheerdegen commented:
Could you rebase on `development` and force push? This contains a bunch of commits that have been merged already.

FYI, your commits aren't signed. See https://github.com/ACCESS-NRI/dev-docs/issues/23 for how to configure this.

heidinett commented on README.md:
```diff
@@ -11,72 +11,28 @@ https://access-hive.org.au/
 - Bold can be used to highlight some words (please do not overuse it);
-- Links should be coded using HTML syntax.<br>External links (links that go to a website other than https://access-hive.org.au) need to specify `target="_blank"` (see HTML cheatsheet below);
-- To signal an important note within the text, consider using a "warning note" (see HTML cheatsheet below);
+- All types of admonitions (info, warning, etc.), collapsible and not, can be used as per [documentation](https://squidfunk.github.io/mkdocs-material/reference/admonitions/), but they are rendered slightly differently. If you need to use the HTML version of them, please follow the HTML cheatsheet below.
```
Edit: "... can be used as described in the [documentation]..."
Edit: "... use the HTML version of these, please refer to the HTML..."

heidinett commented on docs/about/user_support/index.md:
```diff
@@ -6,32 +6,32 @@ On this page we answer frequently asked questions (FAQs), and also give addition
     <p align="center"><img align="center" width="50%" src="/assets/access_hive_links.png" alt="Diagram showing how ACCESS-NRI and the Climate Research Working Groups engage with each other through the ACCESS-Hive via the Access-Hive portal for documentation and the Hive Forum for exchange and support."></p>
 
-??? abstract "What is the difference between _model_, _model component_, and _model configuration_?"
+??? question "What is the difference between _model_, _model component_, and _model configuration_?"
```
Edit: remove comma before 3rd item in list: "model component_ and _model configuration_?"

heidinett commented on docs/models/run-a-model/run-access-om.md:
```diff
@@ -175,117 +214,116 @@ _S_ indicates the status of your run, where:
+This section describes how to modify {{ model }} configuration.<br>
+The modifications discussed in this section can change the way {{ model }} is run by _Payu_, or how its specific [model components] are configured and coupled together.
+
+The `config.yaml` file located in the _control_ directory, is the _Master Configuration_ file, which controls the general model configuration.<br>
```
Remove comma: "... config.yaml file located in the control directory is the Master Configuration file..."

heidinett commented on docs/models/run-a-model/run-access-om.md:
```diff
@@ -175,117 +214,116 @@ _S_ indicates the status of your run, where:
+Symbolic links to these directories are also provided in the _control_ directory for convenience.
 
-## Modifying an {{ model }} configuration
+If a model run is unsuccessful the `work` directory is left untouched to facilitate the identification of the cause of the model failure.
```
Add comma: "If a model run is unsuccessful, the `work` directory... "

heidinett commented on docs/models/run-a-model/run-access-om.md:
```diff
@@ -175,117 +214,116 @@ _S_ indicates the status of your run, where:
+The modifications discussed in this section can change the way {{ model }} is run by _Payu_, or how its specific [model components] are configured and coupled together.
+
+The `config.yaml` file located in the _control_ directory, is the _Master Configuration_ file, which controls the general model configuration.<br>
+It contains several parts, some of which it is more likely will need modification, and others which are rarely changed without having a deep understanding of how the model is configured.
```
I would remove the line break before this sentence as "It" relates to the previous sentence.

heidinett commented on docs/models/run-a-model/run-access-om.md:
```diff
@@ -175,117 +214,116 @@ _S_ indicates the status of your run, where:
 
-One of the most common changes is to adjust the duration of the model run. For example when debugging changes to a model, it is common to reduce the run length to minimise resource consumption and return faster feedback on changes.
+One of the most common changes is to adjust the duration of the model run.<br>
+For example, when debugging changes to a model it is common to reduce the run length to minimise resource consumption and return faster feedback on changes.
```
Add comma: "... when debugging changes to a model, it is common... "

heidinett commented on docs/models/run-a-model/run-access-om.md:
```diff
@@ -295,18 +333,21 @@ To change the run length, edit the `restart_period` field in the `&date_manager_
+    restart_period = 1, 4, 10
 
+!!! warning
+    While `restart_period` can be reduced, it should not be increased to more than 5 years, to avoid errors.
```
Remove comma: "... more than 5 years to avoid errors."

heidinett commented on docs/models/run-a-model/run-access-om.md:
```diff
@@ -295,18 +333,21 @@ To change the run length, edit the `restart_period` field in the `&date_manager_
 ### Modify PBS resources
 
-If the model has been altered and needs longer to complete, more memory, or you want to change which queue it uses then this is the part of `config.yaml` you need to modify:
+If the model has been altered and needs longer time to complete, more memory, or needs to be submitted under a different NCI project, you will need to modify the following section in the `config.yaml`:
```
Replace "longer" with "mor":  "... altered and needs more time to complete..."

atteggiani commented:
> Could you rebase on `development` and force push? This contains a bunch of commits that have been merged already.

Yeah that is because I initially created the branch starting from your branch that was then merged already.
I ran `git rebase development` and then force pushing the content. However, it seems there are still your merged commits.
Not sure how to solve this (other than createing a new branch from `development` and re-apply all the modifications I made on this)

heidinett reviewed (approved):
Looks great! Only suggested a few minor edits - see comments.

atteggiani commented:
> Could you rebase on `development` and force push? This contains a bunch of commits that have been merged already.
> 
> FYI, your commits aren't signed. See [ACCESS-NRI/dev-docs#23](https://github.com/ACCESS-NRI/dev-docs/issues/23) for how to configure this.

@aidanheerdegen 
Done! 
Thank you for pointing out about the signed commits!

atteggiani commented:
References also (almost entirely) written in markdown.

atteggiani commented:
Do we also need to change anything in the [ACCESS-OM](https://access-hive.org.au/models/configurations/access-om/) Models section after the new ACCESS-OM2 configurations release?

flicj191 reviewed (approved):
Looks great, great work!

aidanheerdegen commented on docs/models/run-a-model/run-access-om.md:
````diff
@@ -417,102 +463,107 @@ Each submodel contains additional configuration options that are read in when th
 
 ```yaml
 userscripts:
     error: resub.sh
````
```suggestion
   error: tools/resub.sh
```
Moved scripts to `tools` subdirectories

aidanheerdegen reviewed (changes requested):
Thanks Davide. You've picked up some errors/omissions and the modifications of the text improve readability.

Most of the suggested changes are highlighting styling inconsistencies, removing html elements or making the instructions consistent with updates in the released configs.

atteggiani commented on docs/models/run-a-model/run-access-om.md:
```diff
@@ -80,93 +103,109 @@ The next step is to clone this branch to a location on _Gadi_:
 This separates the small text configuration files from the larger binary outputs and inputs. In this way, the _control_ directory can be in the `$HOME` directory (as it is the only filesystem actively backed-up on _Gadi_). The quotas for `$HOME` are low and strict, which limits what can be stored there, so it is not suitable for larger files.
 
-The _laboratory_ directory is a shared space for all `payu` experiments using the same model.  
+The _laboratory_ directory is a shared space for all *payu* experiments using the same model.<br>
```
*payu* (\*payu*) is not bold, is italic.
**payu** (\*\*payu**) is bold.

Anyway I changed the italic markdown syntax to underscores (\_payu_) to avoid confusion.

atteggiani commented on README.md:
```diff
@@ -11,78 +11,47 @@ https://access-hive.org.au/
-- Code lines/blocks need to be used for lines/blocks of code and terminal commands;
-- Italic needs to be used when referring to specific proper nouns (for example *Gadi* or *Accessdev*);
+- Code lines/blocks need to be used for lines/blocks of code, terminal commands and file/directory paths/names;
+- Italic needs to be used when referring to specific proper nouns (for example *Gadi* or *payu*);
```
payu (and Gadi) are italic here. 
Anyway, I changed the Markdown syntax for italic to undesrcore-wrapped words so it's not confused with the double-start-wrapped bold syntax.

atteggiani commented on docs/models/run-a-model/run-access-om.md:
````diff
@@ -352,58 +393,63 @@ sync:
+This can either be a number (in which case every _nth_ restart file is retained), or a [pandas-style datetime frequency](https://business-science.github.io/pytimetk/guides/03_pandas_frequency.html).<br>
+For example, to preserve the ability to restart {{ model }} every 50 model-years, set:
 ```yaml
 restart_freq: 50Y
````
Are they required to be set as a string or you are just setting it as strings in the released configs? 
Because if the former is true, then we might need to make it clearer in the instructions.

aidanheerdegen replied:
Actually I'm not sure. @jo-basevi?

jo-basevi replied:
The quotes are not needed as it will still be read as a string when loading the yaml file. `Y` is not a supported time unit however (as I think it was tricky with leap-days of what is exactly a year period). Instead there is `YS` which saves the restart at the start of the year?

jo-basevi replied:
Sorry I am not able to link to `restart_freq` in payu's documentation directly but it is under the Model section: https://payu.readthedocs.io/en/latest/config.html#model

atteggiani commented:
Thanks @aidanheerdegen for the review. 

I have updated the branch with your suggestions.

aidanheerdegen reviewed (changes requested):
Just noticed line 103 (`cd 1deg_jra55_ryf`) is missing a prompt (needs `data=input`).

I also realised there is no instructions on how to get help. A link to the user support page (https://access-hive.org.au/about/user_support/) would be appropriate.

These are the form of words I used in the release notice. I think something like this is also worth including as well as the link above:

> If you have questions about [access-om2](https://forum.access-hive.org.au/tag/access-om2) create a topic in the [COSIMA](https://forum.access-hive.org.au/c/cosima/29) category on the [ACCESS Hive Forum](https://forum.access-hive.org.au) and tag it with [access-om2](https://forum.access-hive.org.au/tag/access-om2). If you require assistance [follow the guidelines for requesting help from ACCESS-NRI ](https://forum.access-hive.org.au/t/access-help-and-support/908).

atteggiani commented:
> Just noticed line 103 (`cd 1deg_jra55_ryf`) is missing a prompt (needs `data=input`).
> 
> I also realised there is no instructions on how to get help. A link to the user support page (https://access-hive.org.au/about/user_support/) would be appropriate.
> 
> These are the form of words I used in the release notice. I think something like this is also worth including as well as the link above:
> 
> > If you have questions about [access-om2](https://forum.access-hive.org.au/tag/access-om2) create a topic in the [COSIMA](https://forum.access-hive.org.au/c/cosima/29) category on the [ACCESS Hive Forum](https://forum.access-hive.org.au) and tag it with [access-om2](https://forum.access-hive.org.au/tag/access-om2). If you require assistance [follow the guidelines for requesting help from ACCESS-NRI ](https://forum.access-hive.org.au/t/access-help-and-support/908).

Fixed vitual terminal prompt, and added 'how to get help' section at the end.
@aidanheerdegen

aidanheerdegen reviewed (approved):
LGTM. Thanks Davide.
